demos/interactive_face_detection_demo/cpp_gapi/main.cpp

```diff
@@ -21,10 +21,142 @@
+    if (FLAGS_h || 1 == argc) {
+        std::cout <<   "\t[ -h]                                         " << h_msg
+                  << "\n\t[--help]                                           print help on all arguments"
+                  << "\n\t  -m <MODEL FILE>                             " << m_msg
+                  << "\n\t[--bb_enlarge_coef <NUMBER>]                  " << bb_enlarge_coef_msg
+                  << "\n\t[ -d <DEVICE>]                                " << d_msg
+                  << "\n\t[--dag <DEVICE>]                              " << dag_msg
+                  << "\n\t[--dam <DEVICE>]                              " << dam_msg
+                  << "\n\t[--dem <DEVICE>]                              " << dem_msg
+                  << "\n\t[--dhp <DEVICE>]                              " << dhp_msg
+                  << "\n\t[--dlm <DEVICE>]                              " << dlm_msg
+                  << "\n\t[--dx_coef <NUMBER>]                          " << dx_coef_msg
+                  << "\n\t[--dy_coef <NUMBER>]                          " << dy_coef_msg
+                  << "\n\t[ -i <INPUT>]                                 " << i_msg
+                  << "\n\t[--lim <NUMBER>]                              " << lim_msg
+                  << "\n\t[--loop]                                      " << loop_msg
+                  << "\n\t[--mag <MODEL FILE>]                          " << mag_msg
+                  << "\n\t[--mam <MODEL FILE>]                          " << mam_msg
+                  << "\n\t[--mem <MODEL FILE>]                          " << mem_msg
+                  << "\n\t[--mhp <MODEL FILE>]                          " << mhp_msg
+                  << "\n\t[--mlm <MODEL FILE>]                          " << mlm_msg
+                  << "\n\t[ -o <OUTPUT>]                                " << o_msg
+                  << "\n\t[ -r]                                         " << r_msg
+                  << "\n\t[--show] ([--noshow])                         " << show_msg
+                  << "\n\t[--show_emotion_bar] ([--noshow_emotion_bar]) " << show_emotion_bar_msg
+                  << "\n\t[--smooth] ([--nosmooth])                     " << smooth_msg
+                  << "\n\t[ -t <NUMBER>]                                " << t_msg
+                  << "\n\t[ -u <DEVICE>]                                " << u_msg
+                  << "\n\tKey bindings:"
+                     "\n\t\tQ, q, Esc - Quit"
+                     "\n\t\tP, p, 0, spacebar - Pause"
+                     "\n\t\tC - average CPU load, D - load distrobution over cores, M - memory usage, H - hide\n";
+        showAvailableDevices();
+        slog::info << *InferenceEngine::GetInferenceEngineVersion() << slog::endl;
+        exit(0);
+    } if (FLAGS_i.empty()) {
+        throw std::invalid_argument("-i <INPUT> can't be empty");
+    } if (FLAGS_m.empty()) {
+        throw std::invalid_argument("-m <MODEL FILE> can't be empty");
```

Comment:
```suggestion
        throw std::invalid_argument{"-i <INPUT> can't be empty"};
    } if (FLAGS_m.empty()) {
        throw std::invalid_argument{"-m <MODEL FILE> can't be empty"};
```
Align brackets with pure cpp